DDDEastAnglia/Views/Shared/_Sponsors.cshtml

```diff
@@ -1,63 +1,11 @@
-﻿<section id="sponsors">
+﻿@model IEnumerable<DDDEastAnglia.Models.SponsorModel>
+<section id="sponsors">
```

Comment:
Can the two sponsors' views now be properly merged, so we get actual reuse of code? :smile: